.github/workflows/master.yml

```diff
@@ -0,0 +1,147 @@
+name: master
+
+on:
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches: 
+      - master
+
+jobs:
+  python35:
+    runs-on: ubuntu-latest
+    env:
+      SPARK_VERSION: 2.3.4
+      PANDAS_VERSION: 0.23.4
+      PYARROW_VERSION: 0.10.0
+      PYTHON_EXECUTABLE: xvfb-run python
+      KOALAS_USAGE_LOGGER: databricks.koalas.usage_logging.usage_logger
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: 1.8
+    # Setup Python 3.5 via `apt-get install` since the dafault Python3.5 from `actions/setup-python`
+    # seems to have some problem with Tkinter, so we should manually install the python3.5-tk.
+    # for this, we should use the Python we manually installed, not the default one from `actions/setup-python`
+    - name: Setup Python 3.5
+      run: |
+        sudo add-apt-repository ppa:deadsnakes/ppa
+        sudo apt-get update
+        sudo apt-get install tk-dev python3.5-tk python3 python3.5 python-pip python3-pip
+        sudo rm /usr/bin/python
+        sudo ln -s /usr/bin/python3.5 /usr/bin/python
+    - name: Using cache
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements-dev.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+    - name: Download dependencies
+      run: ./dev/download_travis_dependencies.sh
+    - name: Install dependencies & Run tests
+      run: |
+        sudo apt-get install xclip python-pytest flake8
```

Comment:
Hm, should we install `python-pytest` from `apt-get`? Why we didn't install it from PIP?

Reply:
Because it was not working with 'not installed' error message when we test `Install dependencies` and `Run tests` separately.

but now we integrated them, so i'm testing again. if not necessary, i'll remove it from apt-get install list.

Reply:
`flake8` too. It's from `requirements-dev.txt`.

Reply:
you're right, so i'm testing both of them